### DISCUSSION TITLE
Missing "New project" option in New Organization page?

### DISCUSSION THREAD

**Discussion by okyanusoz** [community/community#30702] [Other Feature Feedback, Questions, & Ideas]
I think there is a weird bug in GitHub's UI. Here's how to reproduce:

1. Click on the plus sign -> New organization -> Free
2. On the new organization page, click on the plus sign again. Notice that the "New project" option is not available here.

I think this was probably overlooked.

Was this intended?

Screenshots below.

Homepage:
![homepage plus sign](https://user-images.githubusercontent.com/46757266/186505771-39fd8d30-fad3-4c51-9507-5ebe98ed47d2.jpg)

/account/organizations/new
![orgs](https://user-images.githubusercontent.com/46757266/186506200-b4e5f035-4f6c-4180-9be0-03d92d64da07.jpg)

I use Mozilla Firefox 103.

**Comment by jfullstackdev** [comment]
I can reproduce the same thing even if my Orgs are not newly created,

maybe because there is a new Projects other than the classic one, so they are not yet updating the dropdown, but you can still create that at the Org's Projects tab

![image](https://user-images.githubusercontent.com/47092464/186663677-88b0fc06-29a9-4abd-a08d-3bda4d1707b1.png)
